### ISSUE TITLE
question: options.ttl vs cookie.maxAge

### ISSUE THREAD

**Issue by techjeffharris** [chncdcksn/mongoose-session#11]
A person should set these to the same lenght of time, no?

I think is what's happening: 
 * `options.ttl` defines the `expires` attribute to the session document (mongodb cleans up once expired)
 * `cookie.maxAge` defines the time that lapses before the cookie expires (cleans up the cookie once expired)

Which makes me wonder what would happen if they were set to differing values...  for example:

```javascript
var express = require('express');

var mongoose = require('mongoose');
mongoose.connect('mongodb://localhost/db');

var app = express();

app.use(require('express-session')({
    cookie: { maxAge: 3600 }, // 3600ms = 1 minute
    key: 'session',
    secret: 'SUPER SECRET SECRET',
    store: require('mongoose-session')(mongoose, { ttl: 3600 }); // 3600s = 1 hour
}));
```

If I understand properly, the cookie would expire after 1 minute, the session document would expire in 1 hour.  Once that minute passes and the cookie expires upon refresh a new cookie and session document would be generated leaving the stale session document sitting in the collection for the next 59 minutes (when it expires).

# tl;dr

Is there any benefit (aside from flexibility) to setting them to differing values?

I'm probably over-thinking this and should just set them both to the same length of time...unless I want stale session documents for development/debugging I suppose.
